### PR TITLE
AB#3303 -- Updated the French and English translations for the Terms & Conditions page

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Updated the French and English translations for the Terms & Conditions page

### Related Azure Boards Work Items
[AB#3303](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3303)

### Screenshots (if applicable)
n/a

### Checklist
- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [x] I have checked that all e2e tests pass by running `npm run test:e2e`

### Test Instructions
View T&C apply page to confirm it conforms with the May release final figma design

### Additional Notes
n/a

### CODE REVIEW

File: frontend/public/locales/en/apply.json [hunk 1]
```diff
@@ -424,7 +415,7 @@
       },
       "how-we-protect-your-privacy": {
         "heading": "How we protect your privacy",
-        "personal-information-collection-and-use-authorization": "The collection and use of your personal information is authorized under the <i>Privacy Act</i>, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws.",
+        "personal-information-collection-and-use-authorization": "The collection and use of your personal information is authorized under the <i>Privacy Act</i>, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws. ",
```
Comment (gregory-j-baker) [inline frontend/public/locales/en/apply.json:419]: Was the added space here intentional?

Comment (gregory-j-baker) [reply frontend/public/locales/en/apply.json:419]: And I think the `<i>` elements should be `<em>`, shouldn't they?

**Review by gregory-j-baker — APPROVED** [review]
Approved with comment.

File: frontend/public/locales/en/apply.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply.json]: Removed spaces, and extended to italics to french translations as well

Comment (gregory-j-baker) [reply frontend/public/locales/en/apply.json]: I guess we can keep the `<i>` for now.. According to the docs, the default set of allowed tags are `['br', 'strong', 'i', 'p']`

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**